source/SIL.AppBuilder.Portal.Frontend/src/data/store.ts

```diff
@@ -1,39 +1,11 @@
-import { Bucket } from '@orbit/core';
```

Comment:
this stuff was all pulled out into a builtin strategy to react-orbitjs, here: https://github.com/developertown/react-orbitjs/blob/master/src/strategies/pessimistic-with-remote-ids.ts